assets/js/googlesitekit/modules/create-info-store.js

```diff
@@ -106,7 +111,7 @@ export const createInfoStore = ( slug, {
 	};
 
 	return {
-		STORE_NAME,
+		storeName,
```

Comment:
Let's preserve the case of the key in the return value here for consistency with other `create*Store` utilities.

Reply:
I've changed this and reverted the tests to use this constant form instead of camel case.